sdks/python/src/honcho/client.py

```
@@ -36,6 +40,26 @@ class Honcho(BaseModel):
     )
     _client: HonchoCore = PrivateAttr()
 
+    @property
+    def core(self) -> HonchoCore:
+        """
+        Access the underlying honcho_core client. The honcho_core client is the raw Stainless-generated client,
```

Pylint (3.3.7)

[convention] Line too long (111/100)

(C0301)